addons/mail/models/mail_link_preview.py

```diff
@@ -3,11 +3,12 @@
 
 from datetime import datetime
 from dateutil.relativedelta import relativedelta
-from lxml import html, etree
+from lxml import html
 from urllib.parse import urlparse
 import requests
 
 from odoo import api, models, fields
+import odoo.addons.mail.tools.link_preview as mail_tools
```

Comment:
why using an alias ?
```suggestion
from odoo.addons.mail.tools import link_preview
```